From 8a55b78aa340810c1a7146f4f6b4084284828f2a Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@protonmail.com>
Date: Wed, 25 Dec 2024 18:48:12 -0700
Subject: [PATCH 1/1] Enable-crypton-connection and related deps

Remove conditional that only allows these newer deps with >=ghc-9.8

Signed-off-by: hololeap <hololeap@protonmail.com>
--- a/darcs.cabal
+++ b/darcs.cabal
@@ -461,7 +461,7 @@ Library
                       exceptions        >= 0.6 && < 0.11,
                       terminal-size     >= 0.3.4 && < 0.4
 
-    if impl(ghc >= 9.8)
+    if true
       cpp-options:    -DHAVE_CRYPTON_CONNECTION
       build-depends:  crypton-connection >= 0.4 && < 0.5,
                       data-default-class >= 0.1.2.0 && < 0.1.3,
-- 
2.45.2

